app/src/main/java/org/gdg/frisbee/android/activity/ParseDeepLinkActivity.java

```diff
@@ -48,12 +51,10 @@ protected void onCreate(Bundle savedInstanceState) {
      * @param deepLinkId The deep-link ID to parse.
      * @return The intent corresponding to the deep-link ID.
      */
-    private Intent parseDeepLinkId(String deepLinkId) {
-        if (deepLinkId == null) {
-            return null;
-        }
+    @Nullable
```

Comment:
Do we need this nullable annotation for return value. It seems that it will never be null.